Sources/MapboxCoreNavigation/EHorizon/RoadObject.swift

```diff
@@ -26,7 +26,7 @@ public struct RoadObject {
     /** `true` if an object is added by user, `false` if it comes from Mapbox service. */
     public let isUserDefined: Bool
     
-    /** `true` if an object is in an urban area, `false` otherwise. */
+    /** This property is set to `nil` if the road object comes from an electronic horizon and location is set to `RoadObject.Location.point(_:)`. */
```

Comment:
Let’s indicate both what the property indicates and the circumstances in which it would be set:

```suggestion
    /**
     Indicates whether the road object is located in an urban area.
     
     This property is set to `nil` if the road object comes from a call to the `RoadObjectStore.roadObject(identifier:)` method and `location` is set to `RoadObject.Location.point(_:)`.
     */
```